src/components/VideoItem.vue

```diff
@@ -207,4 +207,12 @@ export default {
 .shorts-img {
     @apply w-full object-contain;
 }
+
+.video-info {
+    @apply mt-1 text-xs text-gray-600 font-normal;
+}
+
+.video-info .dark {
+    @apply text-gray-300;
+}
```

Comment:
![image](https://github.com/TeamPiped/Piped/assets/20838718/7d36b716-dd60-4e8c-8a51-675695df59cb)
Maybe we should make the text more light for better readability?